decks/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.1.2 on 2022-10-11 01:35
+# Generated by Django 4.1.2 on 2023-01-21 14:42
```

Comment:
While I was in here, I deleted all the migrations and re-created them. Since we are not live anywhere, we can do this. And adding a custom user model mid-stream is a non-trivial task due to how Django does various things.